crates/types/src/net/codec.rs

```diff
@@ -16,6 +16,7 @@ use bytes::Bytes;
 use serde::Serialize;
 use serde::de::DeserializeOwned;
 
+use super::{FromBilrostDto, IntoBilrostDto};
 use crate::protobuf::common::ProtocolVersion;
 
 pub trait WireEncode {
```

Comment:
I wonder if we should make WireEncode be bound to NetSerde types, that would be cool. Can happen in follow-up PRs.